db/migrate/20220103182104_add_categories_to_purchases.rb

```diff
@@ -0,0 +1,10 @@
+class AddCategoriesToPurchases < ActiveRecord::Migration[6.1]
+  def change
+    add_column :purchases, :diapers_money_in_cents, :integer
```

Comment:
Could we add `not null` constraint on these new columns? This way calculations won't go haywire if a column happens to be nil.